firmware/Core/Src/rtos_tasks/rtos_tasks.c

```diff
@@ -121,9 +125,10 @@ void TASK_handle_uart_telecommands(void *argument) {
 
 			// TODO: maybe log/print args too
 			
-			uint8_t args_str_no_parens[end_of_args_idx - start_of_args_idx + 2];
-			memcpy(args_str_no_parens, &latest_tcmd[start_of_args_idx+1], end_of_args_idx - start_of_args_idx - 1);
-			args_str_no_parens[end_of_args_idx - 1] = '\0';
+			const uint8_t arg_len = end_of_args_idx - start_of_args_idx - 1;
```

Comment:
`arg_len` may overflow. `uint8_t` is not long enough.

Reply:
Made uint16, max value is ~65000, I think we won't have an argument that long 😄